test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3136,7 +3136,7 @@ def forward(self, x):
 
     test_name = "test_" + op_name + "_ufunc"
     assert not hasattr(cls, test_name), "{0} already in {1}".format(test_name, cls.__name__)
-    if(op_name in ['erf']):
+    if(op_name in ['erf', 'sinh', 'cosh']):
```

Comment:
This is because of the following errors (`caffe2_onnx_ort1_py3_6_clang7_ubuntu16_04_test`):

```
RuntimeError: No Op registered for Cinh with domain_version of 7
RuntimeError: No Op registered for Cinh with domain_version of 8
RuntimeError: No Op registered for Sinh with domain_version of 7
RuntimeError: No Op registered for Sinh with domain_version of 8
```

My take:

From https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sinh and https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sinh, it states:

> This version of the operator has been available since version 9 of the default ONNX operator set.

I assume the reason it gives segmentation fault for Opset7 and Opset8 is that it does not support opset versions below 9 (https://github.com/pytorch/pytorch/blob/master/test/onnx/test_pytorch_onnx_caffe2.py#L737). This is based on my limited understanding of skipIfUnsupportedMinOpsetVersion from https://github.com/pytorch/pytorch/blob/master/test/onnx/test_pytorch_common.py#L45.